python-sdk/nuscenes/eval/lidarseg/evaluate.py

```diff
@@ -0,0 +1,156 @@
+import argparse
+import json
+import os
+from typing import Dict
+
+import numpy as np
+from tqdm import tqdm
+
+from nuscenes import NuScenes
+from nuscenes.eval.lidarseg.utils import LidarsegChallengeAdaptor, ConfusionMatrix, get_samples_in_eval_set
+
+
+class LidarSegEval:
+    """
+    This is the official nuScenes lidar segmentation evaluation code.
```

Comment:
Should we standardize "nuScenes lidar segmentation" to "nuScenes-lidarseg" everywhere?